src/app/test/service-stubs.ts

```diff
@@ -190,6 +191,22 @@ export class SearchStubService {
     ]);
   }
 
+  initializeToolTips() {
```

Comment:
It's unfortunate this has to be duplicated, but I see code was like that already and probably beyond the scope of this PR.